drivers/ieee802154/Kconfig.rf2xx

```diff
@@ -18,6 +18,19 @@ config IEEE802154_RF2XX_DRV_NAME
 	help
 	  This option sets the driver name
 
+config IEEE802154_RF2XX_NET_IF_NO_AUTO_START
+	bool "RF2X must wait configuration before change to operational mode"
+	default n
```

Comment:
Setting the value to 'n' is not needed as it is the default value. So this does no harm but there is a general guideline not to add these the Kconfig files.

Reply:
I'll fix this, tks!

Reply:
Done!